packages/eslint-plugin/configs/main.js

```diff
@@ -28,5 +28,6 @@ module.exports = {
 		'sitekit/jsdoc-tag-order': [ 'error' ],
 		'sitekit/no-yield-dispatch': [ 'error' ],
 		'sitekit/no-direct-date': [ 'error' ],
+		'sitekit/no-storybook-scenario-label': [ 'error' ],
```

Comment:
We actually don't need to add it here yet as we just want the rule to be enabled in the staged config right now. I gave this a quick test and it works, so I'll remove this and the related "off" above 👍